lib/fizzy/execute.cpp

```diff
@@ -18,6 +18,63 @@ struct LabelContext
 };
 
 
+void match_imported_functions(const std::vector<TypeIdx>& module_imported_types,
+    const std::vector<ImportedFunction>& imported_functions)
+{
+    if (module_imported_types.size() != imported_functions.size())
+        throw std::runtime_error("Module requires " + std::to_string(module_imported_types.size()) +
```

Comment:
This expression (and some similar ones later) should be rather wrapped with `{}`.